components/CryptoSection.tsx

```diff
@@ -21,7 +21,7 @@ declare const window: Window;
 
 interface CryptoSectionProps {
   id: number | string;
-  scAddress: string;
+  smartContractAddress: string;
```

Comment:
@edanfesi in a follow up PR where you introduce both `user_settings` for on/off can you also instead use `metadata` instead of `smartContractAddress`